presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/main/groupdetail/model/GroupEvent.kt

```diff
@@ -13,7 +14,9 @@ fun RecentEventDomainModel.toUiModel(): GroupEvent {
     return GroupEvent(
         id = id,
         title = title.orEmpty(),
-        date = date.toString(),
-        deadline = deadline?.toString().orEmpty(),
+        date = date?.let { LocalDateUtil.format(it) }.orEmpty(),
+        deadline = deadline?.let {
+            LocalDateUtil.format(it, "M월 d일 EEEE hh시 mm분") + " PIC 종료"
```

Comment:
```suggestion
            LocalDateUtil.format(it, "M월 d일 EEEE hh시 mm분 PIC 종료")
```
요거 이렇게 문자열을 합치지 않은 이유가 있는걸까?!

Reply:
뒤에는 뭔가 포맷을 위한 패턴이 아닌 느낌도 들고.. 괜히 넣었다가 내가 모르는 PIC가 존재한다면 어떡하나 이런 걱정에서

Reply:
확인 고마워!! 어푸할게~~